src/author.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+require_relative './item'
+
+class Author < Item
```

Comment:
There should be an option to display all the created authors in the UI but your application seems to miss it. Kindly add this functionality. Every book should have an author, genre, source, and label. However, you are missing all of these things. I have recorded [this video](https://www.loom.com/share/3bb2fbd21a324177b1ba95212895e1bc) where I explained in detail what you need to fix.